src/main/java/org/apache/commons/beanutils2/sql/converters/SqlTimestampConverter.java

```diff
@@ -62,12 +62,6 @@ protected Class<Timestamp> getDefaultType() {
     }
 
     /**
-     * Gets a {@code DateFormat} for the Locale.
```

Comment:
As above.

Reply:
It's already inheriting the JavaDoc from the function it's overriding. I removed it because it didn't look like it needed populating.

![image](https://github.com/apache/commons-beanutils/assets/22801583/feb2320d-fb89-42c7-af21-4aa20716f175)